README.md

```diff
@@ -1,4 +1,4 @@
-# itscalledsoccer
+# itscalledsoccer-js
 
 `itscalledsoccer` is a wrapper around the same API that powers the [American Soccer Analysis app](https://app.americansocceranalysis.com/). It enables Node users to programmatically retrieve advanced analytics for their favorite [MLS](https://en.wikipedia.org/wiki/Major_League_Soccer), [NWSL](https://en.wikipedia.org/wiki/National_Women%27s_Soccer_League), and [USL](https://en.wikipedia.org/wiki/United_Soccer_League) players and teams.
```

Comment:
Just realized the R version has a [different summary](https://github.com/American-Soccer-Analysis/itscalledsoccer-r#overview) than the other two.

Reply:
I think that's alright